plugins/search-backend-module-stack-overflow-collator/alpha-api-report.md

```diff
@@ -0,0 +1,11 @@
+## API Report File for "@backstage/plugin-search-backend-module-stack-overflow-collator"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { BackendFeature } from '@backstage/backend-plugin-api';
```

Comment:
Tbh I don't think this needs to be an alpha export, might as well put it on the main entry point.

Reply:
Good idea, done here: https://github.com/backstage/backstage/pull/20456/commits/39d29a481433cefe6870fd839fc813a4c88d217c 🙌🏻